docs/config.adoc

```diff
@@ -1157,6 +1157,29 @@ This can make it a little confusing for how to set the `x, y` values
 to get the positions that you want.
 Experimentation will be needed.
 
+[[mouse-speed]]
+==== Set the speed of mouse movements
+<<table-of-contents,Back to ToC>>
+
+The action `movemouse-speed` modifies the speed at which `movemouse` and
+`movemouse-accel` function at runtime. It does this by expanding or shrinking
+`min_distance` and `max_distance` while the action key is pressed.
+
+This action accepts one number (unit: percentage, no '%') by which the
```

Comment:
```suggestion
This action accepts one number (unit: percentage) by which the
```

Upon first reading, the suggested removal part confused me so I think it's clearer without it. The example is sufficient to showcase that % should be omitted in my correct.